CMR/python/cmr/search/common.py

```diff
@@ -157,18 +158,20 @@ def _cmr_basic_url(base: str, query: dict, config: dict = None):
         config: configurations, responds to:
             * env - sit, uat, ops, prod, production, or blank for production
     """
-    expanded = ""
-    if query is not None and len(query)>0:
-        expanded = "?" + net.expand_query_to_parameters(query)
+    base = common.always(base, str)
 
-    config = common.always(config)
-    env = config.get('env', '').lower().strip()
-    if len(env)>0 and not env.endswith("."):
-        env += "."
-    if env in ['', 'ops', 'prod', 'production']:
-        env = ""
+    query = common.always(query)
+    query = '' if len(query)<1 else f'?{net.expand_query_to_parameters(query)}'
```

Comment:
similar comment to above, nit about spacing around `<` here

Reply:
I will fix this in another ticket, I needed to merge right away. Sorry.